modular_skyrat/modules/expeditionary-corps/code/jobs.dm

```diff
@@ -0,0 +1,299 @@
+/datum/job/expedition_commander
+	title = "Expedition Сommander"
+	description = ""
+	department_head = list(JOB_QUARTERMASTER)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the quartermaster"
+	selection_color = "#d7b088"
+	//minimal_player_age = 40
```

Comment:
```suggestion
```
There's no precedent for using this, and while I think it'd be interesting it'd likely need a discussion of its own.

Reply:
I kept it just in case.